### PR TITLE
New Topics added for Cloud Network and Network Event Broker tools

### PR DESCRIPTION
**Pull request by dsoumay**
Hi,
I have added two new topics (2 new pages), and images under the Managing Network Configuration topic in the Administration guide. Please review the changes and approve them if they look good.

Thanks,
Soumay

### CODE REVIEW

File: content/en/docs/administration-guide/managing-network-configuration/using-network-event-broker.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+	
+	[Network]
+	Links="ens33 ens37"
+	RoutingPolicyRules="ens33 ens37"
+	UseDNS="true"
+	UseDomain="true"
+
+<br />
+
+	❯ systemctl status network-broker.service
```
Comment (ssahani) [inline content/en/docs/administration-guide/managing-network-configuration/using-network-event-broker.md:83]: please drop it

File: content/en/docs/administration-guide/managing-network-configuration/using-network-event-broker.md [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+ 	                          VARIANT "s" {
+  	                                 STRING "degraded";
+ 	                          };
+ 	                  };
+ 	          };
+ 	          ARRAY "s" {
+ 	          };
+ 	  };
+
+
```
Comment (ssahani) [inline content/en/docs/administration-guide/managing-network-configuration/using-network-event-broker.md:122]: till here. These are not required for documentation

**Review by ssahani — CHANGES_REQUESTED** [review]

**Comment by mikeroySoft** [comment]
layout wise it looks okay... we can merge once we have the changes that @ssahani proposed.

File: content/en/docs/administration-guide/managing-network-configuration/configuring-a-secondary-network-interface-using-cloud-network.md [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+
+`cloud-network` automates the whole manual process of configuring the secondary network interface. It configures the network parameters required for any network interfaces that you create or add to the linux instance. In a cloud environment, instances are set to public IPs and private IPs. If you add more than one private IP for the secondary network interface, the IP other than the one provided by DHCP cannot be fetched and configured for your virtual machine. The `cloud-network` project is designed to adapt the cloud-network environments such as Azure, GCP, and Amazon EC2. `cloud-network` fetches the metadata from the metadata server endpoint, parses the metadata, and then assigns IPs and routes. When `cloud-network` is installed, it automatically configures network interfaces in the cloud frameworks. It detects the available instances using netlink. Additionally, for all the interfaces, including the primary one, it looks for any secondary IP addresses from the metadata server endpoint and configures them on the interface, if any. 
+
```
Comment (ssahani) [inline content/en/docs/administration-guide/managing-network-configuration/configuring-a-secondary-network-interface-using-cloud-network.md:9]: It detects the available instances using netlink. -> It detects the available interfaces using netlink.

**Comment by ssahani** [comment]
LGTM .  Probably squash the last 2 commits  @mikeroySoft  ?

**Pull request merged**